code/modules/economy/supply_packs.dm

```diff
@@ -621,6 +639,15 @@ ABSTRACT_TYPE(/datum/supply_packs)
 	containertype = /obj/storage/crate
 	containername = "Assorted Glowsticks Crate - 4 pack"
 
+/datum/supply_packs/portable_fueltank
+	name = "Portable Welding Fuel Tank"
+	desc = "A single transportable fuel tank, for when you're on the move."
+	category = "Basic Materials"
+	contains = list(/obj/item/reagent_containers/food/drinks/fueltank )
```

Comment:
```suggestion
	contains = list(/obj/item/reagent_containers/food/drinks/fueltank)
```
the smallest of gripes

Reply:
Consistency is important! One sec, will fix!